doc/developer/README.md

```diff
@@ -0,0 +1,33 @@
+![hitobito logo](https://user-images.githubusercontent.com/9592347/184715060-351453d4-d066-4ff6-8f82-95d3b524b62f.svg)
+
+# Developer documentation 👨‍💻
+
+These documents describe different aspects that need to be considered when developing.
+
+## New here?
+[Getting started](01_getting_started.md)
+
+## Intro into modules
+Hitobito is divided into several modules which hold different logic. If you are searching
+for a specific module or feature take a look 🔍 at the list below.
+
+* Common:
+  * [In general](modules/common/common.md)
```

Comment:
```suggestion
  * [In general](modules/common/README.md)
```
maybe use parent title here that always points to README.md ... in this case: modules/common/README.md